packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -361,15 +386,15 @@ class _ExpansionTileState extends State<ExpansionTile> with SingleTickerProvider
 
   Widget _buildChildren(BuildContext context, Widget? child) {
     final ExpansionTileThemeData expansionTileTheme = ExpansionTileTheme.of(context);
-    final Color borderSideColor = _borderColor.value ?? Colors.transparent;
+    final ShapeBorder expansionTileBorder = _border.value ?? const Border();
 
     return Container(
-      decoration: BoxDecoration(
-        color: _backgroundColor.value ?? expansionTileTheme.backgroundColor ?? Colors.transparent,
-        border: Border(
-          top: BorderSide(color: borderSideColor),
-          bottom: BorderSide(color: borderSideColor),
-        ),
+      clipBehavior: Clip.antiAlias,
```

Comment:
Why is this clip added?

Reply:
- without clip
<img width="318" alt="image" src="https://user-images.githubusercontent.com/58727124/190303408-8c4fd015-dab4-44fa-b3f7-072b0e936ad7.png">

- with Clip
<img width="318" alt="image" src="https://user-images.githubusercontent.com/58727124/190303497-64494f5b-674c-49cf-bb28-5c2515560cdc.png">

Reply:
should I ask the user for the `clipBehavior` value?
or can I use `Clip.antiAlias` as the default `clipBehaviour`

Reply:
What would happen now in this case? Would there be a clip if the content overflowed? This might break current users that have overflowing content.

Reply:
maybe we could make it as parameter